src/NovelRT/Maths/GeoBounds.cpp

```diff
@@ -28,7 +28,7 @@ namespace NovelRT::Maths {
     auto minB = otherBounds.position() - otherBounds.getExtents();
     auto maxB = otherBounds.position() + otherBounds.getExtents();
 
-    auto result = minA > maxB | minB > maxA;
+    auto result = (minA > maxB) | (minB > maxA);
```

Comment:
Shouldn't this be short-circuiting? why is this bitwise? Not your fault I know but this is hurting my brain. Can you make this a logical OR and re-run the tests? ty!

Reply:
Changed it. The bitwise OR was indeed the one triggering that parentheses warning, its not needed but will leave it in.

Reply:
Sure thing, thanks!